Tests/DatadogTests/Datadog/DatadogCore/LaunchTimeReaderTests.swift

```diff
@@ -0,0 +1,83 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import Datadog
+
+// TODO: RUMM-2034 Remove this flag once we have a host application for tests
+#if !os(tvOS)
```

Comment:
Just checking: is this `TODO` still relevant? The https://github.com/DataDog/dd-sdk-ios/pull/774 was already done.

Reply:
Ah indeed, good catch :)